barretenberg/cpp/src/barretenberg/world_state/world_state.cpp

```diff
@@ -506,7 +506,15 @@ WorldStateStatusFull WorldState::sync_block(
 
     {
         auto& wrapper = std::get<TreeWithStore<NullifierTree>>(fork->_trees.at(MerkleTreeId::NULLIFIER_TREE));
-        NullifierTree::AddCompletionCallback completion = [&](const auto&) -> void { signal.signal_decrement(); };
+        NullifierTree::AddCompletionCallback completion = [&](const auto& resp) -> void {
```

Comment:
This change is unrelated, but I noticed this was incorrect.

Reply:
Oh that's me, thanks for fixing!